File: codeforces/gyms/100200.asc04/F.php
<?php
if (getenv('USER') == 'watashi') {
  $STDIN = STDIN;
} else {
  $STDIN = fopen('positive.in', 'r');
  $STDOUT = fopen('positive.out', 'w');
  ob_start(function($buffer) {
    global $STDOUT;
    fwrite($STDOUT, $buffer);
  });
}

$n = fgets($STDIN) + 0;
$a = fgets($STDIN);
foreach (explode(' ', $a) as $i) {
  if ($i < 0) {
    $n -= 2;
  }
}
echo max($n, 0), "\n";

File: codeforces/gyms/100200.asc04/F.c
#include <stdio.h>
#include <string.h>

int main(void) {
  int n, i;
  static char s[1 << 20];

#ifndef __WATASHI__
  freopen("positive.in", "r", stdin);
  freopen("positive.out", "w", stdout);
#endif
  scanf("%d ", &n);
  fgets(s, sizeof(s), stdin);
  for (i = 0; s[i] != '\0'; ++i) {
    if (s[i] == '-') {
      n -= 2;
    }
  }
  if (n < 0) {
    n = 0;
  }
  printf("%d\n", n);

  return 0;
}
